pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -34,7 +34,59 @@
 
 class EarlyStopping(Callback):
     r"""
-    Monitor a metric and stop training when it stops improving.
+    Monitor a metric and stop training when a certain validation metric stops improving.
```

Comment:
the "metric" and the "certain validation metric" is the same. 
it is only one metric we monitor and it does not have to be a metric from validation. 

the previous text was more accurate imo.